#! /bin/sh /usr/share/dpatch/dpatch-run
## 13-install-netsvcs.dpatch by Thomas Girard <thomas.g.girard@free.fr>
##
## DP: mark ace_netsvcs and and libnetsvcs as installable

@DPATCH@
--- ACE_wrappers.orig/netsvcs/lib/lib.mpc	2010-07-13 13:10:19.000000000 +0200
+++ ACE_wrappers/netsvcs/lib/lib.mpc	2010-12-28 20:47:37.843219230 +0100
@@ -1,7 +1,7 @@
 // -*- MPC -*-
 // $Id: lib.mpc 91073 2010-07-13 11:10:19Z johnnyw $
 
-project(netsvcs): ace_output, acelib {
+project(netsvcs): ace_output, acelib, install {
   avoids       += ace_for_tao
   sharedname    = netsvcs
   dynamicflags += ACE_BUILD_SVC_DLL
--- ACE_wrappers.orig/netsvcs/servers/servers.mpc	2010-08-26 15:08:43.000000000 +0200
+++ ACE_wrappers/netsvcs/servers/servers.mpc	2010-12-28 20:36:20.435133215 +0100
@@ -1,7 +1,8 @@
 // -*- MPC -*-
 // $Id: servers.mpc 91489 2010-08-26 13:08:43Z johnnyw $
 
-project(Netsvcs_server) : aceexe, avoids_ace_for_tao {
+project(Netsvcs_server) : aceexe, avoids_ace_for_tao, install {
+  install = $(ACE_ROOT)/bin
   exename  = ace_netsvcs
   libs     += netsvcs
   after    += netsvcs
